.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -22,6 +33,8 @@ To check if any of the top-level crates need version increments, go to the zebra
 Zebra follows [semantic versioning](https://semver.org).
 
 Semantic versions look like: `MAJOR`.`MINOR`.`PATCH[`-`TAG`.`PRE-RELEASE]`
```

Comment:
It seems this format is not correct (or not the one we are using). Can we adjust it while we are here?

Reply:
I think the format is correct, but the highlighting is confusing:

```suggestion
Semantic versions look like: MAJOR`.`MINOR`.`PATCH[`-`TAG`.`PRE-RELEASE]
```

Reply:
There is a more detailed explanation inside the `<details>` block, which shows examples of each of the components.